src/uct/api/uct.h

```diff
@@ -385,6 +385,16 @@ enum {
                                                sockaddr */
 };
 
+/*
+ * @ingroup UCT_MD
+ * @brief  Memory types
+ */
+typedef enum {
+    UCT_MD_MEM_TYPE_DEFAULT = 0,   /**< Default system memory */
+    UCT_MD_MEM_TYPE_CUDA,          /**< NVIDIA CUDA memory */
+    UCT_MD_MEM_TYPE_LAST    = UCT_MD_MEM_TYPE_CUDA
```

Comment:
"LAST" should be right after CUDA (which would generate CUDA+1) rather than =CUDA
this way it would be possible to create arrays which entry for each memory type e.g 
`size_t max_zcopy[UCT_MD_MEM_TYPE_LAST]`